src/filter_tree/filter_tree.h

```diff
@@ -166,7 +166,7 @@ void FilterTree_Print
 // components possible following the two rules above
 FT_FilterNode **FilterTree_SubTrees
 (
-	FT_FilterNode *root
+	const FT_FilterNode *root
```

Comment:
👍 
Although note that this function is dangerous as it returns an array to pointers pointing back to the original `root` structure.